src/js/task.js

```diff
@@ -102,16 +102,16 @@ export default class Todolist {
     this.taskDtata.forEach((element, index) => {
       element.index = index;
     });
-    localStorage.setItem('TODOLISTDB', JSON.stringify(this.taskDtata));
-    this.display();
+    // localStorage.setItem('TODOLISTDB', JSON.stringify(this.taskDtata));
+    // this.display();
```

Comment:
Please make sure to remove all the commented code 😉. This will make your project more clear and easier to maintain. Also, kindly notice that by commenting these code pieces your project stopped working as expected.